messages/teleporter/message_manager.go

```diff
@@ -34,6 +34,7 @@ type messageManager struct {
 	// The cache is keyed by the Warp message ID, NOT the Teleporter message ID
 	teleporterMessageCache *cache.LRU[ids.ID, *TeleporterMessage]
 	destinationClients     map[ids.ID]vms.DestinationClient
+	supportedDestinationss map[ids.ID]bool
```

Comment:
Is the extra "s" at the end intentional?

Reply:
Typo 🙂 . Fixed now.